ee/tabby-webserver/src/service/auth.rs

```diff
@@ -356,15 +354,23 @@ impl AuthenticationService for DbConn {
             }
             user
         } else {
-            let Some(invitation) = self.get_invitation_by_email(&email).await? else {
+            let Ok(invitation) = get_or_request_invitation(&self, &email).await else {
```

Comment:
in oauth's case, since the email is verified by 3p, no need to create an invitation? Maybe create a method in as with `create_user`